composer.json

```
@@ -15,7 +15,7 @@
         "symfony-cmf/routing": "^1.2 || ^2.0",
         "symfony/options-resolver": "^2.8 || ^3.3 || ^4.0",
         "symfony/config": "^2.8 || ^3.3 || ^4.0",
-        "jms/metadata": "^1.5"
+        "jms/metadata": "^2.1"
```

could we keep allowing jms metadata 1.5 for older versions?
```suggestion
        "jms/metadata": "^1.5|^2.1"
```